### PR TITLE
fix(ui): login button alignment fixed

### LINKED ISSUE #175: Login button alignment issue
### Description
login button is not perfectly aligned with input elements

### Current situation 
image

### Expected result
image

### PR DESCRIPTION
**Pull request by soham4abc**
## Description

Login button alignment fixed as mentioned in #175 

### Changes

Changed `index.jsx`
Fixes #175 

## Desktop View
image

## Mobile View
image

PTAL @GMishx

### CODE REVIEW

**Comment by soham4abc** [comment]
@GMishx  Do I need to push the updated snapshots?

**Comment by soham4abc** [comment]
PTAL @GMishx

File: src/pages/Home/index.jsx [hunk 1]
```diff
@@ -173,23 +173,30 @@ const Home = ({ location }) => {
...
-                        aria-hidden="true"
-                      />
-                    ) : (
-                      "Login"
-                    )}
-                  </Button>
+                  <Form.Group as={Row}>
+                    <Form.Label column sm="4">
+                      &nbsp;
+                    </Form.Label>
```
Comment (GMishx) [inline src/pages/Home/index.jsx:179]: Is this `Label` needed here?

**Comment by soham4abc** [comment]
@GMishx  Does this require any changes??

File: src/pages/Home/index.jsx (hunk 1 above)
Comment (GMishx) [reply src/pages/Home/index.jsx:179]: I think that is causing the button to go down in mobile view.

Comment (soham4abc) [reply src/pages/Home/index.jsx:179]: Okay I am looking into it

Comment (soham4abc) [reply src/pages/Home/index.jsx:179]: If I remove the mentioned block  this is the case

Desktop/Laptop view:
image


Mobile View:
image


Do let me know if this is acceptable or we need further changes
@GMishx

Comment (soham4abc) [reply src/pages/Home/index.jsx]: @GMishx  Please do have a look 

Mobile view: 
image


Desktop view:
image

Comment (GMishx) [reply src/pages/Home/index.jsx]: Looks better now :+1:

**Comment by GMishx** [comment]
@soham4abc can you please squash your commits and change the title based on [contributing guidelines](https://github.com/fossology/FOSSologyUI/blob/ef3493cf68070634e2007d731aca711b5eb2cdf8/CONTRIBUTING.md#git-commit-conventions)?

**Comment by soham4abc** [comment]
> @soham4abc can you please squash your commits and change the title based on [contributing guidelines](https://github.com/fossology/FOSSologyUI/blob/ef3493cf68070634e2007d731aca711b5eb2cdf8/CONTRIBUTING.md#git-commit-conventions)?

Sure... It's done please have a look

**Comment by GMishx** [comment]
Can you change the commit message as well?

`fix(ui): login button alignment fixed` would be better

**Comment by soham4abc** [comment]
> Can you change the commit message as well?
> 
> `fix(ui): login button alignment fixed` would be better

PTAL @GMishx

**Comment by GMishx** [comment]
Looks like you created PR on wrong branch. It is causing merge conflicts.

Kindly rebase with latest `main` and do a force push.

**Comment by soham4abc** [comment]
PTAL @GMishx  Sorry, Should have gone through the updated Contribution guidelines. Thank you for pointing out my mistake.

**Pull request merged**